src/library_browser.js

```diff
@@ -1225,7 +1225,7 @@ var LibraryBrowser = {
   emscripten_set_window_title__proxy: 'sync',
   emscripten_set_window_title__sig: 'vi',
   emscripten_set_window_title: function(title) {
-    setWindowTitle(AsciiToString(title));
+    setWindowTitle(UTF8ToString(title));
```

Comment:
Do you need to change the above `writeAsciiToMemory` to `stringToUTF8`  ?   Did you verify the test passes locally?

Reply:
this currently throws an assertion, so yeah likely needs this change... testing...